rbac/management/workspace/model.py

```diff
@@ -38,3 +38,14 @@ class Workspace(TenantAwareModel):
 
     class Meta:
         ordering = ["name", "modified"]
+
+    def ancestors(self):
+        """Return a list of ancestors for a Workspace instance."""
+        ancestors = []
+        parent = self.parent
```

Comment:
if there will performance implications with this it looks we can also leverage  recursive queries which Alec  mentioned recently, something like: (and then query workspaces objects)

```
WITH RECURSIVE ancestors AS (
    SELECT uuid, parent_id
    FROM management_workspace
    WHERE uuid = 'f503650a-c72e-446d-825d-39beb9e75eff' <- get ancestors for this workspaces 
    UNION ALL
    SELECT w.uuid, w.parent_id
    FROM management_workspace w
             INNER JOIN ancestors a ON a.uuid = w.parent_id
)
SELECT
    parent_id
FROM
    ancestors WHERE parent_id IS NOT NULL;
```

Reply:
@lpichler I ended up implementing similar to what you suggested here in [9061c6c](https://github.com/RedHatInsights/insights-rbac/pull/1189/commits/9061c6c870d0342e77379f09b70aaed9ebe34e21) with a few changes outlined in the commit/query you should be able to see.